docs/ethereum_attestation.md

```diff
@@ -0,0 +1,133 @@
+# RFC: Ethereum attestation
+
+* Author: @CodeSandwich
+* Date: 2021-03-05
+* Status: draft
+* Community discussion: n/a
+
+## Motivation
+
+The attestation between Link and Ethereum is a valuable building block for a user identity.
+It brings the Link reputation coming from projects and contributions to the
+Ethereum world of DAOs and donations, where it's important to know who's behind an address.
+On the other hand, it lends Ethereum account reputation with its assets and undeniable history
+to Link to build user's trust in the identity.
+
+## Overview
+
+This RFC is built on top of the [identities spec][identities].
+It introduces support for Ethereum address claims on Link
+and a smart contract on Ethereum to make Link identity claims on Ethereum.
+
+## Link identity JSON extension
+
+The identity doc `payload` structure supports a new key: `https://radicle.xyz/ethereum/claim/v1`.
+It can be used only in person identities.
+Under this key there is stored an Ethereum address claim in this format:
```

Comment:
Might be good to rephrase to transmit that it _can_ be stored an Ethereum address.

```suggestion
Under this key, an Ethereum address claim can be stored with this format:
```

Reply:
It seems more ambiguous, the claim MUST be stored and the format MUST be obeyed. I'm afraid of losing that clarity.

Reply:
hm. when I read the current sentence, I read that a claim must exist, whereas claiming an Ethereum address is an optional extension. That's what my suggestion tries to clarify.

Reply:
Ok, what about:

> Under this key, an Ethereum address claim is stored in this format:

Reply:
Perfect, fixed